frontend/src/hooks/api/actions/useFeatureApi/useFeatureApi.ts

```diff
@@ -223,6 +224,26 @@ const useFeatureApi = () => {
         }
     };
 
+    const overrideVariantsInEnvironments = async (
+        projectId: string,
+        featureId: string,
+        variants: IFeatureVariant[],
+        environments: string[]
+    ) => {
+        const put = `api/admin/projects/${projectId}/features/${featureId}/variants-batch`;
+        const req = createRequest(put, {
+            method: 'PUT',
+            body: JSON.stringify({ variants, environments }),
+        });
+
+        try {
+            const res = await makeRequest(req.caller, req.id);
+            return res;
+        } catch (e) {
+            throw e;
+        }
+    };
```

Comment:
I know that this is copy pasted from the other methods, and we always do it like this, me included. Not sure about the original reasoning but the fact is that this could optionally be simplified.

Reply:
I believe the try catch throwing an exception without doing anything is pointless... I could just replace:
```typescript
        try {
            const res = await makeRequest(req.caller, req.id);
            return res;
        } catch (e) {
            throw e;
        }
```
with 
```typescript
        return makeRequest(req.caller, req.id);
```
and now there's no need to `await` because it's the return statement. 

But it's specified that way in [this ADR](https://github.com/Unleash/unleash/blob/main/website/docs/contributing/ADRs/front-end/preferred-data-mutation-method.md).  Maybe the intent of the ADR was to say that you should do something with the exception? Or perhaps it's something we carried away from another place...  :call_me_hand: @FredrikOseberg who may have more context